CHANGELOG.md

```diff
@@ -1,3 +1,141 @@
+# v1.10.0 (2022-10-10)
+
+## OS Changes
+* Automated reconciliation of boot settings ([#2375])
+* Support for static IP addressing ([#2204], [#2330], [#2445])
+* Add sources for NVIDIA's 515 driver ([#2455])
+* Set mode for tmpfs mounts ([#2473])
+* Increase inotify default limits ([#2335])
+* Increase `vm.max_map_count` to match AL2's configurations ([#2344])
+* Add support for configuring DNS settings ([#2353])
+* Migrate `netdog` from `serde_xml_rs` to `quick-xml` ([#2311])
+* Support versioning for `net.toml` ([#2281])
+* Update admin and control container ([#2471], [#2472])
+
+## Orchestrator Changes
+
+### Kubernetes
+
+* Add support for Kubernetes 1.24 variants ([#2437])
+* Remove variant aws-k8s-1.19 ([#2316])
+* Increase the kube-api-server QPS from 5/10 to 10/20 ([#2436], thanks @tzneal)
+* Update eni-max-pods with new instance types ([#2416])
+* Add setting to change `kubelet`'s log level ([#2460], [#2470])
+
+## Platform Changes
+
+### AWS
+* Disable drivers for USB-attached network interfaces ([#2328])
+
+### Metal
+* Add driver support for Solarflare, Pensando, Myricom, Huawei, Emulex, Chelsio, Broadcom, AMD and Intel 10G+ network cards ([#2379])
+
+## Build Changes
+* Extend `external-files` to vendor go modules ([#2378], [#2403], [#2430])
+* Make `net_config` unit tests reusable across versions ([#2385])
+* Add diff-kernel-config to identify kernel config changes ([#2368])
+* Extended support for variants in buildsys ([#2339])
+* Clarify crossbeam license ([#2447])
+* Switch back to `simplelog` in `pubsys` ([#2454])
+* Honor `BUILDSYS_ARCH` and `BUILDSYS_VARIANT` env variables when set ([#2425])
+* Use architecture specific json payloads in unit tests ([#2367], [#2363])
+* Add unified `check` target in `Makefile.toml` for review readiness ([#2384])
+* Add `cargo make` tasks for testing ECS variants ([#2348])
+* Add `cargo make` tasks to perform migration testing in AWS K8s ([#2273])
```

Comment:
I'd like to move this under the Kubernetes news, and the ECS ones to the ECS section.

```suggestion
* Add `cargo make` tasks to perform migration testing for Kubernetes variants in AWS ([#2273])
```